ariadne/explorer/templates/graphiql.html

```diff
@@ -0,0 +1,124 @@
+<!--
+ *  Copyright (c) 2021 GraphQL Contributors
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the license found in the
+ *  LICENSE file in the root directory of this source tree.
+-->
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>{{ title }}</title>
+  <style>
+    body {
+      height: 100%;
+      margin: 0;
+      width: 100%;
+      overflow: hidden;
+    }
+
+    #graphiql {
+      height: 100vh;
+    }
+  </style>
+
+  <!--
+      This GraphiQL example depends on Promise and fetch, which are available in
+      modern browsers, but can be "polyfilled" for older browsers.
+      GraphiQL itself depends on React DOM.
+      If you do not want to rely on a CDN, you can host these files locally or
+      include them directly in your favored resource bundler.
+    -->
+  <script crossorigin src="https://unpkg.com/react@17/umd/react.development.js"></script>
+  <script crossorigin src="https://unpkg.com/react-dom@17/umd/react-dom.development.js"></script>
+
+  <!--
+      These two files can be found in the npm module, however you may wish to
+      copy them directly into your environment, or perhaps include them in your
+      favored resource bundler.
+     -->
+  <link rel="stylesheet" href="https://unpkg.com/graphiql/graphiql.min.css" />
+  {% if enable_explorer_plugin %}
+  <link rel="stylesheet" href="https://unpkg.com/@graphiql/plugin-explorer/dist/style.css" />
+  {% endif %}
+</head>
+
+<body>
+  <div id="graphiql">Loading {{ title }}...</div>
+  <script src="https://unpkg.com/graphiql/graphiql.min.js" type="application/javascript"></script>
+  {% if enable_explorer_plugin %}
+  <script crossorigin src="https://unpkg.com/@graphiql/plugin-explorer/dist/graphiql-plugin-explorer.umd.js" type="application/javascript"></script>
+  {% endif %}
+  <script type="application/javascript">
+    var fetcher = GraphiQL.createFetcher({
+      url: 'https://swapi-graphql.netlify.app/.netlify/functions/index',
+    });
+    var defaultQuery = "# Welcome to GraphiQL\
+#\
+# GraphiQL is an in -browser tool for writing, validating, and\
+# testing GraphQL queries.\
+#\
+# Type queries into this side of the screen, and you will see intelligent\
+# typeaheads aware of the current GraphQL type schema and live syntax and\
+# validation errors highlighted within the text.\
+#\
+# GraphQL queries typically start with a \"{\" character.Lines that start\
+# with a # are ignored.\
+#\
+# An example GraphQL query might look like:\
+#\
+#     {\
+#       field(arg: \"value\") {\
+#         subField\
+#\
+        }\
+#\
+      }\
+#\
+# Keyboard shortcuts:\
+#\
+#   Prettify query: Shift - Ctrl - P(or press the prettify button)\
+#\
+#  Merge fragments: Shift - Ctrl - M(or press the merge button)\
+#\
+#        Run Query: Ctrl - Enter(or press the play button)\
+#\
+#    Auto Complete: Ctrl - Space(or just start typing)\
+#";
```

Comment:
This string does not contain line breaks. It should probably be a template literal instead of containing backslashes (see strawberry: https://github.com/strawberry-graphql/strawberry/blob/c54c0e2922e39b1be1c0134abbb9277fa28a845f/strawberry/static/graphiql.html#L82-L108)